x-pack/solutions/security/plugins/security_solution/public/management/pages/index.tsx

```diff
@@ -80,11 +81,10 @@ const ResponseActionsTelemetry = () => (
   </TrackApplicationView>
 );
 
-const NotesTelemetry = () => (
-  <TrackApplicationView viewId={SecurityPageName.notes}>
+const Notes = () => (
+  <SecurityRoutePageWrapper pageName={SecurityPageName.notes} redirectOnMissing>
```

Comment:
So no more telemetry being capture for this Notes page?

Reply:
Also note that we don't use the `SecurityRoutePageWrapper>` anywhere inside of the `management` page. From what I recall (this is going a while back) we opted to not use it because there were some global styles applied to it in order to support the timeline overlay that was messing with the EUI styles).

Reply:
Thanks @paul-tavares. `SecurityRoutePageWrapper ` actually does include [telemetry](https://github.com/elastic/kibana/blob/10692211bc7fb078d05ad0499d4cf333fbbfc4b8/x-pack/solutions/security/plugins/security_solution/public/common/components/security_route_page_wrapper/index.tsx#L90) within it. The notes page was recently moved within `administration/` sub-links, but as I can see from [recent changes](https://github.com/elastic/kibana/pull/217890) to `SecurityRoutePageWrapper` there are no global styles applied. 

cc @PhilippeOberti @semd.

Reply:
Hey @paul-tavares, thanks for the context.
Just to clarify, the `SecurityRoutePageWrapper` component has been around for about two years and doesn’t apply any global or local styles. Its main purpose is to simplify the conditional rendering logic of route components based on their link configuration (e.g., license, capabilities, experimentalFlags, advancedSettings, etc.).
It also includes telemetry and `spyRoute` support (optionally) for convenience, but styling isn't part of its responsibilities.

It’s possible it’s being confused with `SecuritySolutionPageWrapper`, which includes styling.